### PR TITLE
Use arnested/go-version-action

### LINKED ISSUE #5: Use go-version-action
The [go-version-action](https://github.com/marketplace/actions/go-version-action) provides means to use the Go version that is specified in the `go.mod` file.
It is considered the minimum Go version that should be used to build the module.

### PR DESCRIPTION
**Pull request by arnested**
Fixes #5.

### CODE REVIEW

File: .github/workflows/check.yaml [hunk 1]
```diff
@@ -22,20 +29,24 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: ${{ env.GO_VERSION }}
+          go-version: ${{ needs.go-versions.outputs.minimal }}
```
Comment (EdDev) [inline .github/workflows/check.yaml:32]: @arnested , The check execution does not seem to take the right version.
https://github.com/kiagnose/kiagnose/runs/5873546432?check_suite_focus=true#step:3:7

**Review by EdDev — COMMENTED** [review]
Thank you very much for the contribution!
I think there is something missing at the `lint` job as the go version used at the run looks wrong.

File: .github/workflows/check.yaml (hunk 1 above)
Comment (arnested) [reply .github/workflows/check.yaml:33]: Good catch, @EdDev!

I was wondering how that could happen, and then I realized I forgot the `needs: go-versions` on the  `lint` job.

I have pushed a fix.

**Comment by arnested** [comment]
Good catch, @EdDev!

I was wondering how that could happen, and then I realized I forgot the needs: go-versions on the lint job.

I have pushed a fix.

**Review by EdDev — APPROVED** [review]
Thank you!

**Pull request merged**